src/main/java/cz/xtf/openshift/OpenShiftUtil.java

```diff
@@ -0,0 +1,869 @@
+package cz.xtf.openshift;
+
+import io.fabric8.kubernetes.api.model.*;
+import io.fabric8.kubernetes.api.model.Service;
+import io.fabric8.kubernetes.client.KubernetesClientException;
+import io.fabric8.kubernetes.client.dsl.KubernetesListMixedOperation;
+import io.fabric8.kubernetes.client.dsl.LogWatch;
+import io.fabric8.openshift.api.model.*;
+import io.fabric8.openshift.client.*;
+import lombok.extern.slf4j.Slf4j;
+import rx.Observable;
+import rx.observables.StringObservable;
+
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.*;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+import java.util.function.Function;
+import java.util.function.Supplier;
+import java.util.stream.Collectors;
+
+
+@Slf4j
+public class OpenShiftUtil  implements AutoCloseable {
+	private final NamespacedOpenShiftClient client;
+	private final String namespace;
+
+	public OpenShiftUtil(String masterUrl, String namespace, String username, String password) throws MalformedURLException {
```

Comment:
@Crumby could we add one more constructor to pass OpenShiftConfig object directly. E.g. I'd like to benefit form autoload of `~/.kube/config` or secrets in pod.

Reply:
I have added the possibility, however this config should have set namespace since the utility heavily depends on that.

Reply:
Sure, works for me :) Thanks ☃️